src/sdk/accountspace_fs.js

```diff
@@ -163,8 +156,7 @@ class AccountSpaceFS {
                 const requested_account_encrypted = await nc_mkm.encrypt_access_keys(requested_account);
                 const account_string = JSON.stringify(requested_account_encrypted);
                 nsfs_schema_utils.validate_account_schema(JSON.parse(account_string));
-                await native_fs_utils.update_config_file(this.fs_context, this.accounts_dir,
-                    account_config_path, account_string);
+                await this.configFS.update_account_config_file(params.username, JSON.parse(account_string));
```

Comment:
Why JSON.parse(account_string)? we passed it as a string to `update_config_file`?